tests/blackbox/data/namespace-scoped/kyma_function_syntax_error/scenario.yml

```diff
@@ -0,0 +1,32 @@
+id: kyma-function-syntax-error
+description: The Function has a syntax error in its JavaScript source code.
+user_query: Why is my Kyma Function not working?
+resource:
+  kind: Function
+  api_version: serverless.kyma-project.io/v1alpha2
+  name: my-function-0
+  namespace: kyma-function-syntax-error
+expectations:
+  - name: function_syntax_error
+    statement: points out that the Function has a syntax error in its source code and provides a description of the error.
+    categories:
+      - kyma
+      - problem_finding
+    complexity: 2
+  - name: function_correct_syntax
+    statement: provides an example of how to fix the Function JavaScript source code.
+    categories:
+      - kyma
+      - solution_finding
+    complexity: 2
+  - name: step_by_step_guide
```

Comment:
In [this](https://github.com/kyma-project/kyma-companion/blob/main/tests/blackbox/data/namespace-scoped/kyma_app_serverless_syntax_err/scenario.yml#L10) comparable scenario we rated this criteria with complexity 1 and think we have done this for all scenario for the step by step guide. Why do you give this one complexity 2?

Reply:
You are right. Fixed.